### PR TITLE
PUD-1143: Add additional LDUs. Mark inactive ones ready for deactivation in PUD-1142

### PR DESCRIPTION
**Pull request by philsegal**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/ReferenceDataController.kt [hunk 1]
```diff
@@ -70,13 +72,16 @@ enum class LocalDeliveryUnit(val label: String) {
...
   PS_CARDIFF_AND_VALE("PS - Cardiff and Vale"),
+  PS_CARLISLE("PS - Carlisle"),
   PS_CHESTER("PS - Chester"),
   PS_CHORLEY("PS - Chorley"),
   PS_CORNWALL_AND_ISLES_OF_SCILLY("PS - Cornwall and Isles of Scilly"),
   PS_COVENTRY("PS - Coventry"),
   PS_CREWE("PS - Crewe"),
   PS_CROYDON("PS - Croydon"),
-  PS_CUMBRIA("PS - Cumbria"),
+
```
Comment (AndyNewton) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/ReferenceDataController.kt:82]: Blank lines? 🤷‍♂️

File: src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/ReferenceDataController.kt [hunk 2]
```diff
@@ -47,9 +47,11 @@ enum class LocalDeliveryUnit(val label: String) {
   NORTHERN_IRELAND("Northern Ireland"),
   NOT_APPLICABLE("Not applicable"),
   NOT_SPECIFIED("Not specified"),
+  PS_ACCRINGTON("PS - Accrington"),
```
Comment (AndyNewton) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/ReferenceDataController.kt:50]: Any value in adding comments:` // Active from 10 Dec 2021`?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/ReferenceDataController.kt [hunk 3]
```diff
@@ -88,9 +93,9 @@ enum class LocalDeliveryUnit(val label: String) {
   PS_DYFED_POWYS("PS - Dyfed-Powys"),
   PS_EALING("PS - Ealing"),
   PS_EAST_BERKSHIRE("PS - East Berkshire"),
-  PS_EAST_CHESHIRE("PS - East Cheshire"),
+  PS_EAST_CHESHIRE("PS - East Cheshire"), // No longer active 10 Dec
```
Comment (AndyNewton) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/ReferenceDataController.kt:96]: Implicit `2021`.  Nice ;-)

**Review by AndyNewton — APPROVED** [review]
Couple of QQs.

File: src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/ReferenceDataController.kt (hunk 2 above)
Comment (philsegal) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/ReferenceDataController.kt:50]: yeh could do

**Review by AndyNewton — APPROVED** [review]

**Pull request merged**